backend/src/agents/validator_agent.py

```diff
@@ -0,0 +1,42 @@
+from src.utils import call_model
+from src.agents import Agent, agent_metadata
+
+validator_prompt = """

```

Comment:
Are we adding all prompts as jinja2 templates? I understand it doesn't take any parameters so would be imported just as  a large string but would potentially be cleaner than having it here as a string

Reply:
a very good point. Yeah I'll move it to a jinga template, I agree it would be neater